base/process/process_castanets.cc

```diff
@@ -0,0 +1,394 @@
+// Copyright 2019 Samsung electronics. All rights reserved.
```

Comment:
It seems that this source file is forked from process_posix.cc, why didn't you add your change to process_posix.cc?
My concern is that we will need additional source files for another OS types as well, such as process_castanets_win.cc.

Reply:
About five methods in process_posix.cc require changes by Castanets. As you mentioned, process_castanets_win.cc may be required too. I'm not sure which is a better way - to add #if codes into existing files or to add new *_castanets_* files.

Reply:
@younghajung @hyowon What could be the advantage/disadvantage of adding a new file for each platform?

Reply:
If there are too many Castanets specific code in a class(file), I think creating a separate file would be slightly better. In this case, it is not quite necessary. I will update PR without adding a new file.